File: frontends/go-sh/testdata/t68_case_glob.go
// t68_case_glob: pattern dispatch via if/else
// diagnostics: program prints its result to stdout
s := "hello"
if strings.HasPrefix(s, "h") {
    fmt.Println("star")
} else if strings.Contains(s, "l") || strings.Contains(s, "x") {
    fmt.Println("alt")
}
s = "axl"
if strings.HasPrefix(s, "h") {
    fmt.Println("star2")
} else if strings.Contains(s, "l") || strings.Contains(s, "x") {
    fmt.Println("alt2")
}

File: frontends/perl-sh-go/testdata/t68_case_glob.pl
# t68_case_glob: pattern dispatch via if/elsif
# diagnostics: program prints its result to stdout
my $s = "hello";
if ($s =~ /^h/) {
    print "star\n";
} elsif ($s =~ /l/ || $s =~ /x/) {
    print "alt\n";
}
$s = "axl";
if ($s =~ /^h/) {
    print "star2\n";
} elsif ($s =~ /l/ || $s =~ /x/) {
    print "alt2\n";
}
